tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_kernel_module_loading/rule_audit_rules_kernel_module_loading/syscalls_multiple_per_arg.pass.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_C2S
+# remediation = bash
+
+# Use auditctl, on RHEL7, default is to use augenrules
+sed -i "s%^ExecStartPost=.*%ExecStartPost=-/sbin/auditctl%" /usr/lib/systemd/system/auditd.service
+
+rm -f /etc/audit/rules.d/*
+
+# cut out irrelevant rules for this test
+sed '1,10d' test_audit.rules > /etc/audit/audit.rules
+sed -i '5,8d' /etc/audit/audit.rules
+cat /etc/audit/audit.rules
+true
```

Comment:
What's the purpose of this line and line above (`cat` call) in test?

Reply:
Well noted, they are leftovers of `sed` testing.
I have removed them.